build.sbt

```diff
@@ -29,8 +29,8 @@ val confluentAvroSerializerVersion = "5.0.3"
 val silencer = {
   val Version = "1.4.4"
   Seq(
-    compilerPlugin("com.github.ghik" % "silencer-plugin" % Version cross CrossVersion.full),
-    "com.github.ghik" % "silencer-lib" % Version % Provided cross CrossVersion.full
+    compilerPlugin("com.github.ghik" % "silencer-plugin" % Version cross CrossVersion.patch),
+    "com.github.ghik" % "silencer-lib" % Version % Provided cross CrossVersion.patch
```

Comment:
Adapting the changes from https://github.com/akka/akka/pull/28267 to support the community build better.